Debug-end-to-end-tests.md

```diff
@@ -19,87 +17,14 @@ There are many ways to go about debugging an E2E test, but here's one approach:
 
 3. If you don't understand what the error message means, search for it online. Also look through the test code and `core/test/protractor_utils/action.js` to see if the error message (or something like it) is in our test code.
```

Comment:
I think you need to mention `webdriver_utils/action.js` here too. [Before](https://github.com/oppia/oppia-web-developer-docs/pull/7#discussion_r900404545) you had replaced protractor with webdriver, but I think we need both since the error could come from either file, right?